README.md

```diff
@@ -1,2 +1,114 @@
-# application
+# Application
 Generic helm chart for all kind of applications
+
+# Installing the Chart
+
+To install the chart with the release name my-application in namespace test:
+
+    helm repo add stakater https://stakater.github.io/stakater-charts
+    helm repo update
+    helm install my-application stakater/application --namespace test
+
+# Uninstall the Chart
+
+To uninstall the chart:
+
+    helm delete <name-of-the-chart>
+
+# Configuration
```

Comment:
@usamaahmadkhan to make it more readable breakdown the configuration per manifest

Reply:
Normally helm charts are configured using this approach, as single value might be used in multiple files